### DISCUSSION TITLE
Should air battles use rolls?

### DISCUSSION THREAD

**Discussion by WCSumpton** [triplea-game/triplea#11920] [General]
I noticed that when calculating the airAttack/airDefense values that attackRolls/defenseRolls and chooseBestRoll are also used. This is not stated clearly in PoS2. I can see how this could be assumed, since the air battle is an extension of the main battle. I think there should be a separate option airAttackRolls/airDefenseRolls and chooseBestAirRoll. If not, then PoS2 should clearly state the above, or their uses should be removed for airBattle calculations.

Cheers...

**Comment by beelee1** [comment]
@WCSumpton it would also be nice if the option included a different dice number you are playing the game with, so you could use a 12 sided die for air battles in a 6 sided game.
